docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+          name="Availability"
+          onChange={() => setAvailability('tuesday')}
+          value="tuesday"
+        />
+        <RadioGroup.RadioButton
+          checked={availability === 'wednesday'}
+          id="Wednesday"
+          label="Wednesday"
+          subtext="Evening only"
+          name="Availability"
+          onChange={() => setAvailability('wednesday')}
+          value="wednesday"
+        />
+    </RadioGroup>
+  );
+}
+        `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With Image"
+          description="When including images, you can use the `subtext` property to clearly describe the information being presented by the image, or use the image's `alt` text to provide more context."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [artPreference, setArtPreference] = React.useState();
+
+  return (
+    <RadioGroup legend="Pick a placeholder image" id="imageExample">
+      <RadioGroup.RadioButton
+        checked={artPreference === 'coral'}
+        id="coral"
+        label="Coral"
+        subtext="Botanical art in coral and green"
+        image={<Box height={100} width={80}><Image alt="Botanical art in coral and green" src="https://i.ibb.co/7bQQYkX/stock2.jpg" fit="cover" naturalWidth={1} naturalHeight={1}/></Box>}
+
+        name="Art Preference"
+        onChange={() => setArtPreference('coral')}
+        value="coral"
+      />
+      <RadioGroup.RadioButton
+        checked={artPreference === 'blue'}
+        id="blue"
+        label="Blue"
+        subtext="Typography and shoe in blue"
+        image={<Box height={100} width={80}><Image alt="Typography and shoe in blue" src="https://i.ibb.co/jVR29XV/stock5.jpg" fit="cover" naturalWidth={1} naturalHeight={1}/></Box>}
+        name="Art Preference"
+        onChange={() => setArtPreference('blue')}
+        value="blue"
+      />
+      <RadioGroup.RadioButton
+        checked={artPreference === 'green'}
+        id="green"
+        label="Green"
+        subtext="Abstract art in green"
+        image={<Box height={100} width={80}><Image alt="Abstract art in green" src="https://i.ibb.co/FY2MKr5/stock6.jpg" fit="cover" naturalWidth={1} naturalHeight={1}/></Box>}
+        name="Art Preference"
+        onChange={() => setArtPreference('green')}
+        value="green"
+      />
+    </RadioGroup>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With an error"
+          description="Use `errorMessage` to show an error message below the radio options."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [availability, setAvailability] = React.useState();
+
+  return (
+    <RadioGroup legend="Which time slot works best for you?" errorMessage="Please select one" id="VariantWithErrorMessage">
```

Comment:
nit error could disappear with selection
` errorMessage={!availability && "Please select one"}  `

Reply:
I'll leave as is, since we want people to visually see what the error looks like, and there's no way to de-select a radio after it's been checked

Reply:
Agreed that this is fine — we do similar things with error examples on other components